x-pack/plugins/synthetics/common/runtime_types/monitor_management/monitor_types_project.ts

```diff
@@ -17,6 +17,7 @@ export const ProjectMonitorThrottlingConfigCodec = t.interface({
 export const ProjectBrowserMonitorCodec = t.intersection([
   t.interface({
     id: t.string,
+    config_id: t.string,
```

Comment:
This type is the type that is sent to us by the `synthetics_agent`. The synthetics agent doesn't have context of config_id.